packages/components/bucket-policy/create-bucket-policy.tsx

```diff
@@ -128,6 +129,21 @@ const CreateBucketPolicy: React.FC<CreateBucketPolicyProps> = ({
   const { ns = DATA_FEDERATION_NAMESPACE } = match.params;
 
   const { t } = useTranslation();
+  const breadcrumbs = [
+    {
+      name: DATA_FEDERATION,
```

Comment:
Data federation doesn't need translation?

Reply:
@SanjalKatiyar we removed the translation for some reason. I am not getting it now. Do you remember it?